magefile.go

```diff
@@ -463,6 +463,17 @@ func CrossBuildGoDaemon() error {
 	return devtools.CrossBuildGoDaemon()
 }
 
+func PackageAgentBinary() {
```

Comment:
[Blocker]
Please add the doc as it's used when listing the targets with `mage -l`
```suggestion
// PackageAgentBinary lorem ipsum dolor sit amet
func PackageAgentBinary() {
```